lib/util/hud_utility.rb

```diff
@@ -502,26 +148,15 @@ def transgender_gender_queries
   end
 
   # 3.917.1
-  def living_situation(id, reverse = false)
-    map = living_situations
-    _translate map, id, reverse
-  end
-
-  def living_situations
-    # Technically this should exclude 13, 12, 22, 23, 26, 27, 30, 17, 24, 37
-    ::HudLists.living_situation_map
-  end
-
-  # 3.917.2
-  def residence_prior_length_of_stay(id, reverse = false)
-    map = length_of_stays
-
-    _translate map, id, reverse
-  end
+  # Technically this should exclude 13, 12, 22, 23, 26, 27, 30, 17, 24, 37
+  # def living_situation(id, reverse = false)
+  #   map = living_situations
+  #   _translate map, id, reverse
+  # end
```

Comment:
Let's chat.  I'm unclear why we have that comment, and feel like it could probably be removed/ignored.  I like your solution of just having different lists for the different types of living situations.